src/utils/getMessages.ts

```diff
@@ -0,0 +1,27 @@
+import fs from "fs"
```

Comment:
Organization nitpick question - do `getMessages` & `flattenMessages` both need their own files? They both have to do with i18n support (specifically the `gatsby-plugin-intl` plugin).

Reply:
The problem is NOT the Hysterium of ETHERIUM being #blocked, more than the #number#s of "POSSIBLE Ethereum 1&2 tokens", according to CoinBase, keeps rising 100k ETH1(&2.0) tokens since the BEGINNING of this year(ie. 120million ETH 1&2 tokens available, ON January 2022, to 120.1 ETH1&2 tokens IN February 2022, to NOW; Where there ARE 120.8Million ETH1(&2.0) tokens NOW, May 2022, EQUALLING "121.6 TOTAL ETH1&2.0 tokens" Today, but Eth 2.0 is STILL Not even available to use yet!? 
     Appearing as if ETHERIUM 2.0 is EQUALLY being "staked" is a RIP-OFF! Not only that, the increase of NOW 1.8MILLION MORE of Both ETHERIUMS 1&2, is making MY Current ETHERIUM 1 tokens depreciate in value!! 
     ADDING TWICE AS MANY tokens to "Balance out"_"APPEAR" as if ETH1&2 ARE Equal, while the "staked" ETH2.0 CAN'T Even be Touched, is the BIGGEST RIP OFF I HAVE EVER SEEN IN MY LIFE! WHO KEEPS ADDING THESE "NEW TOKENS" CoinBase is reporting, FOR BOTH ETHERIUM 1&2.0?! 

Thank you for your time, 
CAPT David Simonton-Valento, O-3_USAF, AAS, BT-BAS, Sap-Certification